app/population/steps/edit.js

```diff
@@ -7,10 +11,29 @@ module.exports = {
     resetJourney: true,
     skip: true,
     forwardQuery: true,
+    next: [
+      {
+        fn: req => {
```

Comment:
I'd usually prefer to run branching logic off variables, but we don't have any form variables to work with, which is the point of the next two steps.